lib/money.ex

```diff
@@ -389,9 +389,9 @@ defmodule Money do
       "Total: $100.00"
```

Comment:
We need documentation for the new setting after L369
Please put an example of its usage on L380 (which will be a doc test)